style.css

```diff
@@ -510,4 +513,78 @@
     margin: auto;
     border: 0.5px thin #fff;
   }
+
+  .contact-form {
+    padding-bottom: 10%;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    background-color: white;
+    margin-bottom: 17%;
+  }
+
+  .footer {
+    background-color: white;
+    padding-bottom: 2%;
+  }
+
+  .footer .item {
+    color: #505f79;
+  }
+
+  .footer .liner {
+    height: 4px;
+    width: 145px;
+    margin: 0 auto;
+    background: #0d151e;
+    border-radius: 10px;
+  }
+
+  .contact-form p {
+    font-family: 'Inter', serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    width: 90vw;
+    margin: 20% auto 10% auto;
+    text-align: center;
+    flex: 1;
+  }
+
+  .form_details {
+    display: flex;
+    flex-direction: column;
+  }
+
+  input {
+    width: 90%;
+    padding: 2%;
+    margin: 3%;
+    border: 0;
+    border-bottom: 2px solid #dbd8d7;
+  }
+
+  textarea {
+    width: 90%;
+    margin: 3% auto 5% auto;
+    border: none;
+    background-color: #fbf8f7;
+  }
+
+  .btn {
+    align-items: center;
+    width: 35%;
+    margin: auto;
+    height: 8vh;
+    color: #fff;
+    border: none;
+    background: #ff6b00;
+  }
+
+  .btn:hover {
+    border: 2px solid red;
+    cursor: pointer;
```

Comment:
- Great job so far :muscle: Your project is almost complete. There is just something small you have to do to proceed to the next milestone :smile: 
- [ ] I noticed that your button is missing the right button interaction styles that the template you picked provided you with. I have attached a screenshot below :wink: 
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/213021246-406ef105-2fd7-4754-9989-3f9298489598.png)
- Your Design in all of these states
![image](https://user-images.githubusercontent.com/102699958/213021491-fbfb08eb-5b2c-434d-b3ae-d97281c1b56e.png)
- [ ] Please try to stick to the Figma design as much as possible. This includes colors, margins, padding etc. :wink: